src/types/shared.ts

```diff
@@ -1,4 +1,7 @@
 // Generic numeric value stored as a string. Can be parsed via parseInt or parseFloat.
+
+import { SideEffects } from "src";
+
```

Comment:
Where is this being imported from? If the type is used by different files/types (shared), I'd prefer them to be declared in `types/shared` and imported where needed. E.g if spot and shared need a common type, I'd prefer it to be declared in `shared `and imported in `types/spot`